Nim/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿Console.WriteLine("Welcome to Nim");
+Console.WriteLine("Rules are: You may only draw up to 3 matches.");
+
+int matches = 24; //Starting matches
+
+Console.WriteLine("|||||||||||||||||||||||| (24)"); //Presenting total matches
+Console.WriteLine("Player 1: How many matches do you want to draw?");
+
+round:
+tryAgain:
+
+int playerDraw = int.Parse(Console.ReadLine());
```

Comment:
Remember: this code crashes for invalid input. Use `int.TryParse` to get around that.